train.py

```diff
@@ -0,0 +1,34 @@
+from lstm_word_segmentation.word_segmenter import WordSegmenter
+from lstm_word_segmentation.helpers import download_from_gcs
+import argparse
+
+def parser_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--path', help = 'Dataset file on Google Cloud Storage', type = str)
+    parser.add_argument('--language', help = 'Dataset language: Thai/ Burmese', type = str, default = "Thai")
```

Comment:
In the help argument, we can potentially have other languages too. e.g. we have Lao and Khmr now and in future we may have other languages. Maybe make it more like "such as Thai or Burmese". Also minor, but there shouldn't be a space after "/".